tests/modbus/test.rules

```diff
@@ -1,3 +1,5 @@
+alert modbus any any -> any any (msg:"Testing modbus code function"; modbus: access write; sid:6;)
+alert modbus any any -> any any (msg:"Testing modbus code function"; modbus: function 8; sid:7;)
```

Comment:
Another nit: can we have the new signatures be put _after_ the ones that were already there and follow `sid` order? :)

Reply:
Ok I'll make that change